src/app/(main)/layout.tsx

```diff
@@ -14,6 +14,8 @@ import SkipToMain from "src/components/skipToMain/SkipToMain";
 import { LEGAL_DOCUMENTS_QUERY } from "studio/lib/queries/legalDocuments";
 import { LegalDocument } from "studio/lib/payloads/legalDocuments";
 import styles from "./layout.module.css";
+import { BRAND_ASSETS_QUERY } from "../../../studio/lib/queries/brandAssets";
+import { BrandAssets } from "../../../studio/lib/payloads/brandAssets";
 
```

Comment:
Should we organise our imports?

Reply:
A convention for relative or absolute imports would be nice, and sorting I guess :+1:

Reply:
love it, yes